src/main/_docs/integration-guide/integration-continuous-integration.md

```diff
@@ -48,34 +48,50 @@ where the URL in quotes is the Factory URL to be used with the Jenkins job.
 ### Configure Property Files 
 SSH into your Codenvy instance, navigate to `/home/codenvy` and create 3 `.properties` files:
 
-**First file:** `connectors.properties`
+**1. A Connectors File:** `connectors.properties`
 
 ```text  
 jenkins1=jenkins,factory7nfrelk0v8b77fek,http://userName:password@jenkins.codenvy-dev.com:8080,EvgenTestn
 
-[connector-name],[factory-ID],[$protocol://$userName:$password@$jenkinsURL],[jenkins-job-name]\
+[connector-name],[factory-ID],[$protocol://$userName:$password@$jenkinsURL],[jenkins-job-name]
 ```   
 
-**Second file:** `credentials.properties`
+**2. A Credentials File:** `credentials.properties`
 
 ```text  
 username=yourCodenvyUsername
 password=yourCodenvyPassword
-# username and password should be for the user that created a base Factory\
+# username and password should be for the user that created a base Factory
 ```   
 
-**Third file:** `github-webhooks.properties`
+**3. A GitHub or BitBucket Properties File:**
```

Comment:
Currently we do not support integretion of Bitbucket hosted version (bitbucket.org), so I think it would be better to write **3. A GitHub or BitBucket Server Properties File:**